src/ConfigCatHooks.tsx

```diff
@@ -23,6 +19,28 @@ function useFeatureFlag<T extends SettingValue>(key: string, defaultValue: T, us
   return { value: featureFlagValue, loading };
 }
 
+function useFeatureFlag<T extends SettingValue>(key: string, defaultValue: T, user?: User): {
+  value: SettingTypeOf<T>;
+  loading: boolean;
+} {
+  const configCatContext = useContext(ConfigCatContext);
+
+  if (configCatContext === void 0) throw Error("useFeatureFlag hook must be used in ConfigCatProvider!");

```

Comment:
It would be nice to make the error message clearer. E.g.

```suggestion
  if (configCatContext === void 0) throw Error("useFeatureFlag hook must be used in a ConfigCatProvider with no configId/configAlias/whatever!");
```

Reply:
[obsolete] This method was removed.